recipe/meta.yaml

```diff
@@ -25,14 +25,16 @@ requirements:
     - tomli
     - pip
   run:
+    - click >=8.1
+    - rapids-dask-dependency ==23.12.*
```

Comment:
Currently this fails as we don't have `rapids-dask-dependency` in conda-forge. Also there is not a formal 23.12 release of `rapids-dask-dependency` on PyPI (though this may come soon)

For now filed a tracking issue: https://github.com/rapidsai/dask-cuda/issues/1297

Reply:
The upstream issues have been resolved

That said, this package needs to be added to `conda-forge` ( https://github.com/conda-forge/staged-recipes/issues/24770 )